workflow/ignite/create_standard_trainer_validator.py

```diff
@@ -79,3 +82,5 @@ def add_default_event_handlers(
     )
 
     EarlyStopping(model_score_function, trainer, config).attach(validator)
+
+    return trainer, validator
```

Comment:
newline endoffile

Reply:
Need something that does this automatically....

Reply:
an IDE? ;)